config/grafana/config.go

```diff
@@ -107,4 +107,11 @@ func Configure(p *ujconfig.Provider) {
 			Extractor:         SelfPackagePath + ".UserEmailExtractor()",
 		}
 	})
+	p.AddResourceConfigurator("grafana_organization", func(r *ujconfig.Resource) {
+		r.References["organization"] = ujconfig.Reference{
+			TerraformName:     "grafana_organization",
+			RefFieldName:      "OrganizationRef",
+			SelectorFieldName: "OrganizationSelector",
+		}
+	})
```

Comment:
```suggestion
	p.AddResourceConfigurator("grafana_organization", func(r *ujconfig.Resource) {
		r.References["organization"] = ujconfig.Reference{
			TerraformName:     "grafana_organization",
			RefFieldName:      "OrganizationRef",
			SelectorFieldName: "OrganizationSelector",
		}
	})
```

This block does nothing, you can remove it. References refer to attributes of the resource. In this case, it's looking for an `organization` attribute within the `grafana_organization` resource, which does not exist

Reply:
Thanks!